sklearn/decomposition/tests/test_factor_analysis.py

```diff
@@ -13,8 +13,10 @@
 from sklearn.utils.testing import assert_array_almost_equal
 from sklearn.exceptions import ConvergenceWarning
 from sklearn.decomposition import FactorAnalysis
+from sklearn.utils.testing import ignore_warnings
 
 
+@ignore_warnings
```

Comment:
Please do not put `ignore_warnings` without at least adding a comment to explain which warning we expect to be raised and why we can safely ignore it in this test.

Ideally I would rather use `assert_warns` directly inside the code to make it even more explicit that we expect that warning to happen for a specific call.